gensim/scripts/word2vec2tensor.py

```diff
@@ -44,6 +44,7 @@
 import argparse
 
 import gensim
+from smart_open import smart_open
```

Comment:
please add a test for `word2vec2tensor` function (for avoid this issue in future)